src/adagrams.js

```diff
@@ -1,15 +1,132 @@
 export const drawLetters = () => {
-  // Implement this method for wave 1
+  let letter_quantity = {
```

Comment:
Style: This should be `const` since you never reassign `letter_quantity`.  

You modify but you don't _reassign_ it.